torch/utils/_triton.py

```diff
@@ -2,6 +2,19 @@
 import hashlib
 
 
+@functools.lru_cache(None)
+def device_supports_triton_float8() -> bool:
+    import torch
+
+    n = torch.cuda.device_count()
+    if n < 1:
+        return False
+    capabilities = {torch.cuda.get_device_capability(i) for i in range(n)}
+    min_capability = sorted(capabilities)[0]
```

Comment:
just doing `min(list)` should also work, right? so maybe simply `min(torch.cuda.get_device_capability(i) for i in range(n))`?